pkg/driver/command_driver.go

```diff
@@ -71,7 +72,26 @@ func (d *CommandDriver) exec(op *driver.Operation) error {
 	}
 	cmd.Env = pairs
 	cmd.Stdin = bytes.NewBuffer(data)
-	out, err := cmd.CombinedOutput()
-	fmt.Fprintln(op.Out, string(out))
-	return err
+
+	// Make stdout and stderr from driver available immediately
+
+	stdout, _ := cmd.StdoutPipe()
+	go func() {
+		io.Copy(op.Out, stdout)
+	}()
+	stderr, _ := cmd.StderrPipe()
+	go func() {
+		io.Copy(op.Out, stderr)
+	}()
+	err = cmd.Start()
+	if err != nil {
+		return fmt.Errorf("cmd.Start(%s) failed: %v", d.cliName(), err)
+	}
+
+	err = cmd.Wait()
+	if err != nil {
+		return err
+	}
+
+	return nil
```

Comment:
In fact, you can contract the last few lines into the following if you like:
```
return cmd.Wait()
```
Not everybody likes this style though - take your pick.